theano/sandbox/cuda/nnet.py

```diff
@@ -1,5 +1,9 @@
+import theano
 from theano import Op, Apply
+from theano import tensor as T
 from theano.compat.six import StringIO
+from theano.sandbox.cuda.blocksparse import sparse_block_dot_SS
```

Comment:
You should move this under "from theano.sandbox.cuda.basic_ops import as_cuda_ndarray_variable".